docs-v2/guides/infrastructure.mdx

```diff
@@ -4,52 +4,39 @@ sidebarTitle: 'Infrastructure'
 description: 'Learn more about the infrastructure of Nango.'
 ---
 
-<Frame caption="Architecture overview.">
-  <img src="/images/architecture.png"/>
-</Frame>
-
-**API**
-
-The API layer, developed with Node, serves as the front door to Nango, handling requests from customer applications. It processes operations related to integrations, connections, actions, and syncs, offering a unified interface for developers.
-
-**Control plane store**
-
-This Control plane store, running on Postgres, handles configuration data for users, integrations, connections, and integration scripts.
-
-**Jobs queue**
-
-The Jobs queue, running on Postgres, manages the scheduling of script executions (along with the Orchestrator Node service). It’s responsible for orchestrating the timing and execution of data synchronization tasks, ensuring that sync scripts run according to their schedules without loss or delay.
+### Architecture diagram
 
-**Jobs service**
-
-The Jobs Service dispatches jobs to runner services. It acts as the central manager that allocates integration script execution tasks to available runners, balancing load and ensuring efficient processing of integration logic.
-
-**Runner service**
-
-Runner services, running on Node, are isolated per customer and responsible for running the integration scripts. These services directly interact with external APIs, executing the custom logic defined in action, sync, and webhook scripts. When self-hosting Nango, there is only one runner.
-
-**Records store**
-
-This Records storage, running on Postgres, component persists the records by syncs being executed in the runners.
+<Frame caption="Nango's infrastructure components">
+  <img src="/images/nango-architecture.png" />
+</Frame>
```

Comment:
Maybe we could generate it automatically with [Specfy](https://www.specfy.io/) 😭 🤡

Reply:
Would love a demo <3